common/v2/components/AccountList.tsx

```diff
@@ -349,8 +393,28 @@ function buildAccountTable(
     })
     .sort(getSortingFunction(sortingState.activeSort));
 
+  const getColumns = (
+    columnList: (string | JSX.Element)[],
+    deletePresent: boolean,
+    privacyPresent: boolean
+  ) => {
+    switch (`${deletePresent}-${privacyPresent}`) {
+      // Excludes both columns
+      case 'false-false':
+        return columnList.slice(0, columnList.length - 2);
+      // Includes only the delete column, excludes the privacy tag column
+      case 'true-false':
+        return [...columnList.slice(0, columnList.length - 2), columnList[columnList.length - 1]];
+      // Includes only the privacy tag column, excludes the delete column
+      case 'false-true':
+        return columnList.slice(0, columnList.length - 1);
+      // Includes both delete && privacy tag column
+      case 'true-true':
+        return columnList;
+    }
```

Comment:
There has got to be a better way to do this, right?

Reply:
i think it reads easier, which is why i used it during dev. but not a big deal to have it just be
```
case (deletePresent && privacyPresent):
case (deletePresent && !privacyPresent):
case (!deletePresent && privacyPresent):
case (!deletePresent && !privacyPresent):
```